core/src/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -32,90 +32,248 @@
 using Microsoft.Identity.Core;
 using Microsoft.Identity.Core.Cache;
 using System.Collections.ObjectModel;
+using Microsoft.Identity.Core.Helpers;
 
 namespace Microsoft.Identity.Core
 {
     internal class TokenCacheAccessor : ITokenCacheAccessor
     {
-        private const string AccessTokenServiceId = "com.microsoft.identity.client.accessToken";
-        private const string RefreshTokenServiceId = "com.microsoft.identity.client.refreshToken";
-        private const string IdTokenServiceId = "com.microsoft.identity.client.idToken";
-        private const string AccountServiceId = "com.microsoft.identity.client.Account";
+        public const string CacheKeyDelimiter = "-";
+
+        static Dictionary<string, int> AuthorityTypeToAttrType = new Dictionary<string, int>()
+        {
+            {AuthorityType.AAD.ToString(), 1001},
+            {AuthorityType.MSA.ToString(), 1002},
+            {AuthorityType.MSSTS.ToString(), 1003},
+            {AuthorityType.OTHER.ToString(), 1004},
+        };
+
+        enum CredentialAttrType
+        {
+            AccessToken = 2001,
+            RefreshToken = 2002,
+            IdToken = 2003,
+            Password = 2004
+        }
 
         private const bool _defaultSyncSetting = false;
         private const SecAccessible _defaultAccessiblityPolicy = SecAccessible.AfterFirstUnlockThisDeviceOnly;
 
+        private readonly string DefaultKeychainGroup = "com.microsoft.adalcache";
+        private readonly string TeamIdKey = "teamIDHint";
+
+        private string keychainGroup;
+
+        private string GetBundleId()
+        {
+            return NSBundle.MainBundle.BundleIdentifier;
+        }
+   
+        public void SetSecurityGroup(string securityGroup)
```

Comment:
We don't enforce code sytle yet, but we should try to order the code elements as described in the MSFT code style: http://stylecop.soyuz5.com/SA1201.html namely: 

Fields
Constructors
Delegates
Events
Enums
Properties
public Methods
private methods

Reply:
yes we do not, set some meeting were code style enforcing might be discuss

Reply:
By enforcing I meant having StyleCop run as part of the build and potentially breaking the build. Right now the enforcement comes through PRs :). Do you not agree with having a consistent way of ordering elements?

Reply:
pls. take in the code style discussion

---
In reply to: [200084285](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1108#discussion_r200084285) [](ancestors = 200084285)